components/dropbox/dropbox.app.mjs

```diff
@@ -0,0 +1,431 @@
+import dropbox from "dropbox";
+import fetch from "isomorphic-fetch";
+import get from "lodash/get.js";
+import config from "./config.mjs";
+import isString from "lodash/isString.js";
+import isEmpty from "lodash/isEmpty.js";
+
+const Dropbox = dropbox.Dropbox;
+
+export default {
+  type: "app",
+  app: "dropbox",
+  propDefinitions: {
+    dropbox,
+    pathFolder: {
```

Comment:
No need to include `dropbox` as a prop here.

Reply:
Oops, Idk why this is here :)